gdk4/src/lib.rs

```diff
@@ -12,12 +12,11 @@
 #![allow(deprecated)]
 
 pub use ffi;
+extern crate bitflags;
+extern crate cairo;
 extern crate gdk_pixbuf;
 extern crate gio;
-#[macro_use]
 extern crate glib;
-extern crate bitflags;
-extern crate cairo;
 extern crate libc;
 extern crate pango;
```

Comment:
Hrm you shouldn't have any `extern crate` at all afterwards :) They're redundant with Rust 2018

Reply:
Oh I wasn't aware of that, thanks!

Reply:
Just like `use $crate;` it's not needed anymore. Dependencies are always in scope